ansible/roles_studentvm/studentvm_user/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # Account name for the student user
-studentvm_user_name: lab-user
+studentvm_user_user_name: lab-user
```

Comment:
That change was intentional. _user_user_ just doesn't read right.

Reply:
Ok, this was broken because of the defaults referenced other out of date vars. I'll change these back and fix the studentvm config defaults.

Reply:
fixes pushed.